modules/ROOT/pages/bat-install-task.adoc

```diff
@@ -14,7 +14,7 @@ Before installing the BAT CLI, ensure that AdoptOpenJDK's OpenJDK 8 is installed
 . Install Windows Subsystem Linux (WSL)
 +
 The instructions are at https://docs.microsoft.com/en-us/windows/wsl/install-win10.
-. Install Java 8, cURL, and unzip installed in the Linux distribution that you installed in your WSL.
+. Install Java 8 and cURL in the Linux distribution that you installed in your WSL.
```

Comment:
unzip also it' s a tool required to be installed. The correct requirement should be: `Install Java 8, cURL and unzip in the Linux distribution that you installed in your WSL.`